### ISSUE TITLE
[Fixed in GH-6052] Safari: Missing Visual Focus on My Account: Personal Details

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4333]
Nothing has visual focus on this page, strangely not even the select menu, which in other pages does have it. What could be the reason?

![Screen Shot 2019-08-29 at 8 55 51 PM](https://user-images.githubusercontent.com/15113219/63985891-ac400b80-ca9f-11e9-8af9-eb02e83907e2.png)

**Comment by kacperknapik** [comment]
- Chrome: ✔
- Firefox: ✔ (note: focus border is huge, select field has additional dotted border inside)
- Safari: ❓

**Comment by KateChuen** [comment]
Chrome & Firefox: OK

Missing keyboard accessibility & visual focus on the:
- save and cancel buttons on Safari

**Comment by KateChuen** [comment]
Tested on dev3. Missing visual focus on safari for:

- all the input fields
- all the buttons

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 2 31 57 PM](https://user-images.githubusercontent.com/15113219/73568491-7d570880-4436-11ea-8ff7-5e22ad78f06b.png)
![Screen Shot 2020-01-31 at 2 32 09 PM](https://user-images.githubusercontent.com/15113219/73568492-7d570880-4436-11ea-8a9d-4d4cdc960247.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome ✅
- Firefox ✅
- Safari ✅
